code/src/components/Loading.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export const Loading = () => {
+  return (
+    <div class="loader">Loading...</div>
```

Comment:
❗❗ with react components we should use `className`
```suggestion
    <div className="loader">Loading...</div>
```

Reply:
Done